SU-SOE/stanford_soe_helper
# 7.x 2.x soe 2949 soe 2937 collections masthead

# READY FOR REVIEW

# Summary
- This fixes the masthead for the collection node pages.
- This is branched from https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-SOE-2949

# Needed By (Date)
- Sprint end

# Criticality
- Fixes broken stuff


# Steps to Test

- Navigate to /magazine/future-everything
- Verify you the masthead is the same height as on /magazine/collections

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2937

## Related PRs
https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-SOE-2949

## More Information
Bill, I realize now that this did not really need to branch from https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-SOE-2949

You might want to just pull this into your SASS.

## Folks to notify
@kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
